coordinator/core/src/main/kotlin/net/consensys/zkevm/domain/Blob.kt

```diff
@@ -163,3 +163,39 @@ enum class BlobStatus {
   COMPRESSION_PROVING,
   COMPRESSION_PROVEN
 }
+
+data class BlobSubmittedEvent(
+  val blobs: List<BlockInterval>,
+  val endBlockTime: Instant,
+  val lastShnarf: ByteArray,
+  val submissionTimestamp: Instant,
+  val transactionHash: String
```

Comment:
there is an inconsistency here: `transactionHash: String` `lastShnarf: ByteArray` even though they both are kind of hasshes. I suggested to make them all ByteArray.